### ISSUE TITLE
broken/missing and/or inconsistent mobile/responsive header nav across static sites

### ISSUE THREAD

**Issue by samthomson** [status-im/status.im#171]
Created here but actually it relates to all static sites and possibly the shared-partials repo.
On small screens the navigation is inconsistent. Some sites miss it altogether. On some the mobile menu is triggered by clicking 'more', and on some there is a hamburger button.
- How should it appear? (more, hamburger, something else)
- should it be standardise?

Answer the above two and assign me and I'll look at sorting it.

status.im:
![image](https://user-images.githubusercontent.com/6588340/49212716-467e0080-f3f5-11e8-922a-1b3640cb568a.png)
![image](https://user-images.githubusercontent.com/6588340/49212843-95c43100-f3f5-11e8-979c-0286fb32fcf9.png)

incubate:
![image](https://user-images.githubusercontent.com/6588340/49212730-5269c280-f3f5-11e8-9513-2360758e3172.png)
![image](https://user-images.githubusercontent.com/6588340/49212823-893fd880-f3f5-11e8-97d5-265fe762d280.png)

studio:
![image](https://user-images.githubusercontent.com/6588340/49212761-63b2cf00-f3f5-11e8-96a8-64a05519edc1.png)

nimbus:
![image](https://user-images.githubusercontent.com/6588340/49212788-71685480-f3f5-11e8-91f2-de5116ba2cfd.png)
![image](https://user-images.githubusercontent.com/6588340/49212871-a2488980-f3f5-11e8-859a-a1ee0a7c8aad.png)

**Comment by andytudhope** [comment]
Hey @samthomson - I took on this one, just because there was a lot that needed doing behind the scenes.

Due to current market conditions, we are no longer going to pursue Studio actively, so I had to remove it from all relevant places (still need to figure out what to put on the home page in its place). Which means we don't need to worry about that site.

Then, I updated both Nimbus and Incubate. Incubate is not perfect with both menus, but it's a damn site better than it was. I also moved both the community and project popups into the shared folders. Once the build machines are ready (there is a massive queue right now), we should see these changes across all the dev sites, I will merge them all, and close here. Feel free to get started on all the other issues you've highlighted :heart:
